### PR TITLE
feat: Strapi script refactor and bug fix while fetching record

### PR DESCRIPTION
**Pull request by talwinder50**
closes: issue-20

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: cmd/strapi-demo/createdemodata/start.go [hunk 1]
```diff
@@ -20,15 +20,15 @@ import (
...
-	post                  = "post"
-	get                   = "get"
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the strapi-demo instance on. Format: HostName:Port."
+	hostURLEnvKey        = "STRAPI-DEMO_ADMIN_URL"
+	adminURLEndpoint     = "/admin/auth/local/register"
+	studentCardsEndpoint = "/studentcards"
+	transcriptEndpoint   = "/transcripts"
+	post                 = "POST"
```
Comment (talwinder50) [inline cmd/strapi-demo/createdemodata/start.go]: This was the main cause of the failer... post and get were in lower case so nothing was getting posted or retrieved

File: cmd/strapi-demo/createdemodata/start.go [hunk 2]
```diff
@@ -58,11 +58,11 @@ func GetStartCmd() *cobra.Command {
 }
 func createStartCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "create-demo-data",
-		Short: "Create demo data",
+		Use:   "start",
```
Comment (fqutishat) [inline cmd/strapi-demo/createdemodata/start.go]: why change to start

Comment (talwinder50) [reply cmd/strapi-demo/createdemodata/start.go]: all the other api which we run in local have start flag ... i forgot this had different flag so it took me some time to troubleshoot.  So to maintain consistency specially while troubleshooting, i feel we should stick to start

Comment (fqutishat) [reply cmd/strapi-demo/createdemodata/start.go]: but this not server

Comment (fqutishat) [reply cmd/strapi-demo/createdemodata/start.go]: it's different than other projects

Comment (fqutishat) [reply cmd/strapi-demo/createdemodata/start.go]: usually we use start for starting a server

Comment (talwinder50) [reply cmd/strapi-demo/createdemodata/start.go]: ok i will revert give me 5 min

Comment (fqutishat) [reply cmd/strapi-demo/createdemodata/start.go]: There is no server logic inside your start cmd command

Comment (talwinder50) [reply cmd/strapi-demo/createdemodata/start.go]: done

**Pull request merged**